authority-portal-frontend/src/app/shared/common/portal-layout/sidebar/sidebar.component.ts

```diff
@@ -47,13 +47,16 @@ export class SidebarComponent implements OnInit, OnDestroy {
     combineLatest([
       this.globalStateUtils.getDeploymentEnvironment(),
       this.globalStateUtils.userInfo$.pipe(
-        map((it) => it.organizationName),
+        map((it) => ({
+          organizationName: it.organizationName,
+          organizationMdsId: it.organizationMdsId,
+        })),
```

Comment:
it goes just to an internal method, and this code base does not actively use unit tests. we could just pass the user Info